.dockerignore

```diff
@@ -5,7 +5,9 @@
 node_modules
 visualization/data
 visualization/vis-master/data
+visualization/vis-master/dist
```

Comment:
the dist folder is generated and we don't need to send it to the docker container.